content/docs/contributing/docs.md

```diff
@@ -99,8 +99,11 @@ To build the project and run it:
 - `yarn build` - build assets in the `public` directory.
 - `yarn start` - run production static server over the `public` directory.
 
-> All the tests, formatting, and linters below will be enforced automatically
-> upon [submitting PRs](#submitting-changes).
+> All the tests, formatting, and linters below will be automatically enforced
+> before every commit with [Husky](https://typicode.github.io/husky) and
+> [lint-staged](https://github.com/okonet/lint-staged) before each commit, as
+> well as with [GitHub Actions](https://github.com/features/actions) upon
+> [submitting PRs](#submitting-changes).
```

Comment:
Mentioning `husky` and `lint-staged` alongside our PR CI, reassuring the user we automate this as much as possible.